src/server_manager/web_app/app.ts

```diff
@@ -137,10 +124,37 @@ type DigitalOceanSessionFactory = (accessToken: string) => digitalocean_api.Digi
 type DigitalOceanServerRepositoryFactory = (session: digitalocean_api.DigitalOceanSession) =>
     server.ManagedServerRepository;
 
+export class NotificationManager {
+  // @ts-ignore
+  constructor(private toast) {}
```

Comment:
What is the type of this and how do I find out?

Reply:
That's a <paper-toast> element. It's defined in the approot html.

Reply:
Do we have typing information for it? I don't know what type to assign it.